packages/manager/src/features/CloudPulse/shared/CloudPulseResourcesSelect.tsx

```diff
@@ -76,6 +82,18 @@ export const CloudPulseResourcesSelect = React.memo(
       return resources && resources.length > 0 ? resources : [];
     }, [resources]);
 
+    const setResourceLimit = React.useMemo(() => {
```

Comment:
It appears to be a setter function, to be more readable can we use "resourceLimit" here?